neusomatic/python/call.py

```diff
@@ -287,13 +291,19 @@ def pred_vcf_records(ref_file, final_preds, true_path, chroms, vartype_classes,
     pool = multiprocessing.Pool(num_threads)
     try:
         all_vcf_records = pool.map_async(pred_vcf_records_path, map_args).get()
-        all_vcf_records = filter(lambda x: x, all_vcf_records)
         pool.close()
     except Exception as inst:
         logger.error(inst)
         pool.close()
         traceback.print_exc()
         raise Exception
+
+    for o in all_vcf_records:
+        if o is None:
+            raise Exception("pred_vcf_records_path failed!")
+
+    all_vcf_records = filter(lambda x: x, all_vcf_records)
```

Comment:
Is this removing `None` records from the list?

Reply:
that is to remove empty ones

Reply:
It is for empty lists [ ] not None. Do you know easier way to handle that?

Reply:
It will still work. See https://docs.python.org/2/library/functions.html#filter